src/runtime_src/core/edge/drm/zocl/Makefile

```diff
@@ -30,7 +30,7 @@ ifneq ($(cflags_zocl),)
 endif
 
 # Version is specific to release
-XRT_DRIVER_VERSION ?= 2.15.0
+XRT_DRIVER_VERSION ?= 2.14.0
```

Comment:
I guess this should be defined in a top-level file instead of being buried in a lot of files all over the place.
@uday610 @stsoe

Reply:
@keryell this change is for driver, user space uses cmake flow where we have top level CMake file with this info. We build this driver using petalinux which will not have access to top level CMake.

Reply:
Thanks for this information. Then why not explaining this in this file with a comment so that everyone can understand the purpose?

Reply:
sure, I will add a comment. Thanks

Reply:
Thanks. But I was thinking to a comment explaining the root cause you give in your answer. Like:
```suggestion
# The version is repeated here to be used by the device driver in the case of a PetaLinux build
# when the user-space top-level CMake configuration file specifying the version is not available.
XRT_DRIVER_VERSION ?= 2.14.0
```
To be truncated to 80 characters as usual.
Otherwise I will ask again the question about it the next time I read this. ;-)

Reply:
Added the comment explaining that we cant use top level CMake. Made sure characters in line are < 80 : )